### PR TITLE
Change Visual Studio to "no plugin necessary"

### PR DESCRIPTION
**Pull request by treyhunner**
Per @Mpdreamz's [tweet](https://twitter.com/Mpdreamz/status/801170946555985920)

Maybe we should wait until after VS2017 is officially released to merge this.

Also where should this link point to?

### CODE REVIEW

**Comment by ffes** [comment]
I would hold this back until it is officially released.

The feature is announced for the RC in https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes (search for EditorConfig) no anchor to point to.

**Comment by poke** [comment]
Bump. This can be merged now, VS 2017 has been out for a while now.

File: index.html [hunk 1]
```diff
@@ -132,6 +132,7 @@ <h2 id="overview">What is EditorConfig?</h2>
     <li><a href="https://github.com/JetBrains/intellij-community/tree/master/plugins/editorconfig"><img src="logos/rubyMine.png" alt="RubyMine logo"><span>RubyMine</span></a></li>
     <li><a href="https://www.sourcelair.com/features/editorconfig"><img src="logos/sourcelair.png" alt="SourcLair logo"><span>SourcLair</span></a></li>
     <li><a href="https://tortoisegit.org/"><img src="logos/TortoiseGit.png" alt="TortoiseGit logo"><span>TortoiseGit</span></a></li>
+    <li><a href="https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options"><img src="logos/visualstudio-pro.png" alt="Visual Studio Pro logo"><span>Visual Studio Professional</span></a></li>
```
Comment (jedmao) [inline index.html]: It should probably be noted somewhere that this is only version 2017 and above (future).

File: index.html [hunk 2]
```diff
@@ -160,7 +161,6 @@ <h2 id="overview">What is EditorConfig?</h2>
     <li><a href="https://github.com/editorconfig/editorconfig-textadept#readme"><img src="logos/textadept.png" alt="Textadept logo"><span>Textadept</span></a></li>
     <li><a href="https://github.com/Mr0grog/editorconfig-textmate#readme"><img src="logos/textmate.png" alt="TextMate logo"><span>TextMate</span></a></li>
     <li><a href="https://github.com/editorconfig/editorconfig-vim#readme"><img src="logos/vim.png" alt="Vim logo"><span>Vim</span></a></li>
-    <li><a href="https://github.com/editorconfig/editorconfig-visualstudio#readme"><img src="logos/visualstudio-pro.png" alt="Visual Studio Pro logo"><span>Visual Studio Professional</span></a></li>
```
Comment (jedmao) [inline index.html]: And this guy might still be relevant for folks using versions prior to 2017.

File: index.html (hunk 1 above)
Comment (poke) [inline index.html]: (Also, this functionality isn’t limited to the Professional edition, so no idea why that’s mentioned there)

**Comment by jedmao** [comment]
Also, [`insert_final_newline` and `trim_trailing_whitespace` properties are not supported](https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes#KIeditoride).

I think people might still want to download the extension, but the extension should probably be updated to remove the baked-in features of Visual Studio and just handle these two properties ☝️ that aren't supported.

**Comment by treyhunner** [comment]
@jedmao should we link to https://github.com/editorconfig/editorconfig-visualstudio#readme instead of the announcement page and explain the situation there?

**Comment by jedmao** [comment]
@treyhunner that sounds like a solid plan!

**Comment by treyhunner** [comment]
@jedmao updated the link and merged

**Pull request merged**